src/wp-admin/includes/file.php: get_home_path() add hook get_home_pat…

alexrah opened the pull request:
https://core.trac.wordpress.org/ticket/60622

alexrah committed:
src/wp-admin/includes/file.php: get_home_path() add hook get_home_path to return value

alexrah committed:
src/wp-admin/includes/file.php: get_home_path() fix PHP Coding Standards

mukeshpanchal27 commented on src/wp-admin/includes/file.php:
```diff
@@ -117,7 +117,9 @@ function get_home_path() {
-	return str_replace( '\\', '/', $home_path );
+	$home_path = str_replace( '\\', '/', $home_path );
+
+	return apply_filters( 'get_home_path', $home_path );
```
Add inline document for a new filter.

mukeshpanchal27 reviewed (commented):
Thanks @alexrah for the PR! Could you please add document for the filter and unit test?

alexrah committed:
Merge branch 'trunk' into patch-get_home_path

alexrah committed:
Inline documentation for hook get_home_path in wp-admin/includes/file.php

alexrah committed:
src/wp-admin/includes/file.php: get_home_path() remove empty line to fix PHPCS checks

alexrah committed:
tests/phpunit/tests/admin/includesFile.php: test filter get_home_path defined in wp-admin/includes/file.php

alexrah committed:
tests/phpunit/tests/admin/includesFile.php: fix PHPCS

alexrah committed:
tests/phpunit/tests/admin/includesFile.php: fix PHPCS errors related to filter_get_home_path()

alexrah commented:
Added inline documentation for the filter as requested in src/wp-admin/includes/file.php function: get_home_path()
PS: I left the @since line uncompleted 

Added Unit tests in tests/phpunit/tests/admin/includesFile.php method: test_get_home_path()
